miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/closealert/MiniAppCloseAlertInfo.kt

```diff
@@ -0,0 +1,11 @@
+package com.rakuten.tech.mobile.miniapp.closealert
+
+import androidx.annotation.Keep
+
+/** An object to prepare information of close alert popup. */
+@Keep
+data class MiniAppCloseAlertInfo(
+    var shouldDisplay: Boolean = false,
+    var title: String? = "",
+    var description: String? = ""
```

Comment:
I think these don't need to be nullable since you're setting a default, right? We can always set at least an empty string.